src/binary.h

```diff
@@ -86,7 +88,15 @@ enum class NameSectionSubsection {
   Global = 7,
   ElemSegment = 8,
   DataSegment = 9,
-  Last = DataSegment,
+  // tag names are not officiaully part of the extended-name-section proposal
```

Comment:
```suggestion
  // tag names are not officially part of the extended-name-section proposal
```

Reply:
... or we could speculatively just leave this part of the comment out, and propose a change to extended-name-section